tests/selenium/test_bootstrap.py

```diff
@@ -0,0 +1,79 @@
+import dash
+import pytest
+import time
+
+import dash_bootstrap_components as dbc
+import dash_html_components as html
+from dash_table import DataTable
+
+import pandas as pd
+
+url = "https://github.com/plotly/datasets/raw/master/" "26k-consumer-complaints.csv"
+rawDf = pd.read_csv(url)
+df = rawDf.to_dict("rows")
+
+
+def get_app(fixed_rows, fixed_columns, ops):
+    app = dash.Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+    props = dict(
+        id="table",
+        data=df[0:250],
+        columns=[
+            {"name": i, "id": i, "hideable": i == "Complaint ID"} for i in rawDf.columns
+        ],
+        style_table=dict(height="500px", maxHeight="500px", overflow="auto"),
+        editable=True,
+        sort_action="native",
+        include_headers_on_copy_paste=True,
+        **fixed_rows,
+        **fixed_columns,
+        **ops
+    )
+
+    app.layout = html.Div([DataTable(**props)])
+
+    return app
+
+
+@pytest.mark.parametrize(
+    "fixed_rows,fixed_rows_description",
+    [(dict(), "unfixed_rows"), (dict(fixed_rows=dict(headers=True)), "fixed_rows")],
+)
+@pytest.mark.parametrize(
+    "fixed_columns,fixed_columns_description",
+    [
+        (dict(), "unfixed_columns"),
+        (dict(fixed_columns=dict(headers=True)), "fixed_columns"),
+    ],
+)
+@pytest.mark.parametrize(
+    "ops,ops_description",
+    [
+        (dict(), "ops: none"),
+        (dict(row_selectable="single", row_deletable=True), "ops: sinle+deletable"),
+        (dict(row_selectable="multi", row_deletable=True), "ops: multi+deletable"),
+    ],
+)
+def test_tbbs001_display(
+    dash_thread_server,
+    dash_duo,
+    test,
+    fixed_rows,
+    fixed_columns,
+    ops,
+    fixed_rows_description,
+    fixed_columns_description,
+    ops_description,
+):
+    test.start_server(get_app(fixed_rows, fixed_columns, ops))
+
+    test.table("table").is_ready()
+
+    time.sleep(3)
```

Comment:
The table "jiggles" to find the right column width with certain prop values (fixed_*). Adding this delay makes sure it's done "jiggling".

Reply:
Would it be worthwhile including some signal in the DOM, like `data-table-is-iterating` or something, that gets removed when this jiggling is done? That would be useful for dash-docs too, where we see some percy flakiness on these tables - we could just wait until there are no iterating DOM elements.

Reply:
@alexcjohnson Actually I think I'm wrong about this. While at some point I had added a loop to resize the table columns for the fixed scenarios, I've eventually removed it entirely. Will try removing the sleep and see if there's any inconsistencies. I'd like to see the inconsistencies in table rendering you've been seeing. Some recent bug fixes have changed some aspects of rendering and would have caused a slew of `1 to few` pixels alignment changes.

Reply:
Great! Maybe those issues have been resolved - but if I see them again I'll let you know :)